src/Test.UnitTests.Sarif.Multitool.Library/ExportValidationRulesMetadataCommandTests.cs

```diff
@@ -27,15 +27,14 @@ public void BuildRule_GeneratesExpectedMarkdown()
                 { "TextShortDescription.md", new TestRule2() },
             };
 
-            var resourceExtractor = new ResourceExtractor(this.GetType());
+            var resourceExtractor = new TestAssetResourceExtractor(this.GetType());
             foreach (KeyValuePair<string, SarifValidationSkimmerBase> test in tests)
             {
                 var sb = new StringBuilder();
                 var command = new ExportValidationRulesMetadataCommand();
                 command.BuildRule(test.Value, sb);
 
-                string expectedMarkdown = resourceExtractor.GetResourceText(test.Key);
-
+                string expectedMarkdown = resourceExtractor.GetResourceText($"ExpectedOutputs.{test.Key}");
```

Comment:
>GetResourceText [](http://example.com/codeflow?start=60&length=15)

can this  also use the `GetResourceExpectedOutputsText` and remove the `ExpectedOutputs.`

Reply:
Yes! you understand the pattern. :)